packages/mgt-spfx-utils/README.md

```diff
@@ -0,0 +1,155 @@
+# Utilities for SharePoint Framework Webparts using Microsoft Graph Toolkit
+
+[![npm](https://img.shields.io/npm/v/@microsoft/mgt-spfx-utils?style=for-the-badge)](https://www.npmjs.com/package/@microsoft/mgt-spfx-utils)
+
+![SPFx 1.15.2](https://img.shields.io/badge/SPFx-1.15.2-green.svg?style=for-the-badge)
+
+Helper functions to ease lazy loading of mgt components when using disambiguated web components in SharePoint Framework Webparts.
+
+## Installation
+
+To load lazy Microsoft Graph Toolkit components from the library, add the `@microsoft/mgt-spfx-utils` package and if using React, the `@microsoft/mgt-react` package as dependencies to your SharePoint Framework project:
+
+```bash
+npm install @microsoft/mgt-spfx-utils
+```
+
+or
+
+```bash
+yarn add @microsoft/mgt-spfx-utils
+```
+
+or
+
+```bash
+npm install @microsoft/mgt-spfx-utils @microsoft/mgt-react
+```
+
+or
+
+```bash
+yarn add @microsoft/mgt-spfx-utils @microsoft/mgt-react
+```
+
+> **Important:** Since a given web component tag can only be registered once these approaches **must** be used along with the `customElementHelper.withDisambiguation('foo')` approach as this allows developers to create disambiguated tag names.
+
+The use of the disambiguated tag names allows developers to include the MGT components in the webpart bundle to use new versions of MGT without updating the `@microsoft/mgt-spfx` version deployed within a SharePoint environment. The trade-off in doing so is an increased bundle size for the webpart.
```

Comment:
What if we'd rephrase it to:

```suggestion
By disambiguating tag names of Microsoft Graph Toolkit components, developers can use their own version of MGT rather than using the centrally deployed `@microsoft/mgt-spfx` package. This allows them to avoid colliding with other SharePoint Framework components, built by other developers. When disambiguating tag names, MGT is included in the SPFx bundle, increasing its size.
```